content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md

```diff
@@ -2,14 +2,46 @@
 title : "Chainguard Enforce Changelog"
 description: "Chainguard Enforce Changelog"
 type: "article"
-date: 2023-01-10 10:56:36 +0000 UTC
+date: 2023-01-18 18:11:40 +0000 UTC
 draft: false
 images: []
 weight: 799
 ---
 
 ## Introduction
-Any customer facing changes to Chainguard Enforce or [`chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/) are highlighted in the following notes. Any new features, bug fixes, or general ease of use improvements will be listed under the corresponding release version.
+Any customer facing changes to Chainguard Enforce or [`chainctl`](/chainguard/chainguard-enforce/chainctl-docs/how-to-install-chainctl/) are highlighted in the following notes. Any new features, bug fixes, or general ease of use improvements will be listed under the corresponding release version.
+
+### v0.1.56
+Release date: 2023-01-18
+#### Feature
+- Add support for AWS AppRunner services to cluster discovery.
+- Agentless discovery will now attempt to create an SQS queue subscribing to the audit log SNS topic in the region the cluster lives within.
+
+
+### v0.1.55
+Release date: 2023-01-17
+
+Customer facing changes: N/A
```

Comment:
Do we want this on the frontend site?

Reply:
It's not ideal, but that's how it renders from the upstream release notes where there are no changes that impact customers. I think the future version that can merge content from academy and upstream could do away with this.